Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -1360,6 +1369,9 @@ void MediaPlayerPrivateGStreamer::handleMessage(GstMessage* message)
             error = MediaPlayer::NetworkError;
```

Comment:
break command from line 1363 is not changing the logic?

Reply:
seems to yes, I will add sending av reports there too